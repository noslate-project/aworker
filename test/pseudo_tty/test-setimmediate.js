const { setImmediate } = load('timer');

setImmediate(() => { console.log("𒐕𒐕𒐕"); });

console.log("𒐕");

setTimeout(() => {
  console.log("𒐕𒐕");
}, 0);