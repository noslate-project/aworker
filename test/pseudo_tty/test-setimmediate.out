𒐕
𒐕𒐕
𒐕𒐕𒐕